src/plots/plot.ts

```diff
@@ -837,7 +827,7 @@ export class Plot extends Component {
     return {};
   }
 
-  protected static _scaledAccessor<D, R>(binding: Plots.IAccessorScaleBinding<D, R>) {
+  public static _scaledAccessor<D, R>(binding: Plots.IAccessorScaleBinding<D, R>) {
```

Comment:
public _methodName ? we should rename without underscore. or can we get away with protected?